### ISSUE TITLE
Establish one-way video from server to client

### ISSUE THREAD

**Issue by pgaston** [aiortc/aiortc#1043]
My use case is video only from the server to a browser client.   This works fine (thanks!) when I use the example *server* code.

tldr; How can I only publish video from the server to the browser, without asking the user permission to use their camera?

Two challenges:
- The given example (in client.js) calls *navigator.mediaDevices.getUserMedia(constraints)* which will then ask the user for permission to use the camera.    I don't want this - as I'm not using the camera.
- If I set the constraints to be not needed on the browser:
```
    var constraints = {
        audio: false,   // no audio, for now
        video: false     // video yes, but only as a reader, not creator
    };
```
Then I only have a data channel established.    No video.   Not surprisingly, as the code here isn't triggered
```    @pc.on("track")
    def on_track(track):
        logger.info("Track %s received", track.kind)
```

I then manually add  my video track...
```
pc.addTrack(VideoStreamTrack())
```
and I then get a server error on the *offer*:
```
ERROR:aiohttp.server:Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/aiohttp/web_protocol.py", line 452, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.8/dist-packages/aiohttp/web_app.py", line 543, in _handle
    resp = await handler(request)
  File "ros2webrtc.py", line 280, in offer
    await pc.setLocalDescription(answer)
  File "/usr/local/lib/python3.8/dist-packages/aiortc/rtcpeerconnection.py", line 788, in setLocalDescription
    t._currentDirection = and_direction(t.direction, t._offerDirection)
  File "/usr/local/lib/python3.8/dist-packages/aiortc/rtcpeerconnection.py", line 259, in and_direction
    return sdp.DIRECTIONS[sdp.DIRECTIONS.index(a) & sdp.DIRECTIONS.index(b)]
ValueError: None is not in list
```

Thanks!

**Comment by pgaston** [comment]
hmmm, seems like I have a fix... based on SO answer...

in the client: replace the 'getUserMedia block with 
```
    // from https://stackoverflow.com/questions/50002099/webrtc-one-way-video-call
    pc.addTransceiver('video');
    // this step seems to be optional:
    pc.getTransceivers().forEach(t => t.direction = 'recvonly');
    negotiate();
```
and 
```
    // close local audio / video
    pc.getSenders().forEach(function(sender) {
        // with our one way setup, seems like sender.track can be null - makes sense
        if (sender.track) {
            sender.track.stop();
        }
    });

```

and in server.py side...   currently after await pc.setRemoteDescription(offer), but probably anywhere after pc is created
```
    pc.addTrack(VideoStreamTrack())
```

**Comment by pgaston** [comment]
Works fine!

**Comment by Ornstein89** [comment]
@pgaston  Thank you! Saved tons of time!
